src/smt/smt_engine.cpp

```diff
@@ -2817,7 +2804,8 @@ void SmtEnginePrivate::staticLearning() {
 }
 
 // do dumping (before/after any preprocessing pass)
-static void dumpAssertions(const char* key, const AssertionPipeline& assertionList) {
+static void dumpAssertions(const char* key,
+                           const preproc::AssertionPipeline& assertionList) {
```

Comment:
I believe that the `preproc::` here is not necessary since you have a `using namespace CVC4::preproc` above.